custom_model_runner/datarobot_drum/drum/drum.py

```diff
@@ -80,13 +80,27 @@ def __init__(self, runtime):
             get_metadata(self.options)
 
         if self.run_mode in [RunMode.FIT, RunMode.PUSH]:
-            # always populate the validator, even if info isn't provided.  If no info then the validators will be empty
-            # and pass by default.
+            # always populate the validator, even if info isn't provided. Use the default type schema if no
+            # schema is provided, strict validation is enabled, and the target type is transform
+            type_schema = {}
+            use_default_type_schema = False
+            strict_validation = not self.options.disable_strict_validation
+            if self.options.model_config:
+                type_schema = self.options.model_config.get("typeSchema", {})
+
+            if not type_schema and strict_validation and self.target_type == TargetType.TRANSFORM:
+                print(
```

Comment:
decided to not put this behind the verbose flag since it's seems important to convey to the user

Reply:
maybe print to stderr? Also, can you add something which explains what the default is, or where to find this info?

Reply:
should we be using logger.warning instead? @yakov-g

Reply:
logger does not go to stdout. It has to be print/stdout because that's what gets printed to the user

Reply:
In this file, all user-messages are prints

Reply:
And for this info, we should put it in the README for transforms

Reply:
are you sure logger doesn't go to stdout? where does it go?

Reply:
and we should be printing to stderr from inside drum, not stdout.

Reply:
> are you sure logger doesn't go to stdout? where does it go?
I think Jesse means logger goes to whatever it is configured to go, so may not go to stdout.
Currently it is not configured, so by default looks like it is stderr.

Reply:
ok lets use logger then here pls.

Reply:
it won't be shown to the user if it's in logger. Locally, `info`s do not go to stdout

Reply:
If we plan to change logger to go to stdout then we need to evaluate everything we write in there and make sure it looks user friendly and not double log (in a few places we log + print). So until then, the print needs to stay

Reply:
kk...